changefeedtrigger/__init__.py

```diff
@@ -0,0 +1,94 @@
+import time
```

Comment:
I think we shouldn't put that much logic in __init__ file @xiafu-msft correct me if I'm wrong.

This seems to be a place for tiny module definition and anything complex should be delegated to other .py.
https://stackoverflow.com/questions/448271/what-is-init-py-for

Reply:
We had a call and worked together to make some changes. Carmen will post a pr soon!